### PR TITLE
Updated README process

### PR DESCRIPTION
**Pull request by nick-markowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +40,31 @@ GRUB_TBOOT_POLICY_DATA=list.data
...
+        #module /2nd_gen_i5_i7_SINIT_51.BIN
+```
+
+## Enable TBOOT
+1. Reboot
+2. Boot into the tboot kernel. Tboot will fail, but this is expected.
+3. Own the TPM using the well-known SRK password:
+    `tpm_takeownership -z`
+4. Run `./create-lcp-tboot-policy_el<6,7>.sh $tpm_owner_password` to create and install policy
+  * In EL6, adjust `CMD_LINE` head -n based on grub.conf ordering.
```
Comment (jeefberkey) [inline README.md:68]: this can be made better in the script, and will need to be made better if we want puppet to do this

File: README.md [hunk 2]
```diff
@@ -11,34 +11,24 @@ Other sources include (in order of perceived helpfulness):
...
 1. Activate the TPM in the BIOS and set a BIOS password
 2. Ensure VTd is enabled in the BIOS
 3. Boot into the kernel you intend to trust
 4. Install trousers, tpm-tools, tboot and start the tcsd daemon:
-    `yum install -y trousers tpm-tools tboot; systemctl start tcsd`
-5. Own the TPM using the well-known SRK password:
-    `tpm_takeownership -z`
-6. Download the appropriate SINIT for your platform:
-    `https://software.intel.com/en-us/articles/intel-trusted-execution-technology`
+    `yum install -y trousers tpm-tools tboot; systemctl start tcsd; chkconfig tcsd on`
```
Comment (jeefberkey) [inline README.md]: > chkconfig tcsd on

the systemd way to do this is `systemctl enable tcsd` and if you're trying to be backwards compatible `systemctl start tcsd` should be a `service` call

File: README.md [hunk 3]
```diff
@@ -11,34 +11,24 @@ Other sources include (in order of perceived helpfulness):
...
+5. Download the appropriate SINIT for your platform from [Intel](https://software.intel.com/en-us/articles/intel-trusted-execution-technology)
     1. Extract
     2. Copy the `.BIN` file to `/boot`
+6. Modify your grub configuration
+  * There should be no references to list.data or SINIT modules after this step, they should be
+commented out or non-existent.
+  * Your kernel parameters may vary, and you can add/subtract as needed
```
Comment (jeefberkey) [inline README.md]: you have to regenerate the policy to perform a measured launch again

File: README.md [hunk 4]
```diff
@@ -11,34 +11,24 @@ Other sources include (in order of perceived helpfulness):
...
+  * You MUST include `intel_iommu=on`
+  * There MUST NOT be more than one space between kernel parameter options
 
-## EL7
+#### EL7 Grub Configuration
```
Comment (jeefberkey) [inline README.md]: grub2 for more correctness

File: README.md [hunk 5]
```diff
@@ -50,15 +40,31 @@ GRUB_TBOOT_POLICY_DATA=list.data
...
-4. ~~Manually edit `/etc/grub2.cfg`:~~
-    1. ~~Add `--unrestricted` to the tboot entries~~
-    2. ~~Add `module /list.data` in the middle~~
-4. Make sure the SINIT is the last module loaded in the GRUB configuration
-5. Run `./create-lcp-tboot-policy_el7.sh $tpm_owner_password` to create and install policy
-6. Reboot
-7. Select tboot kernel module
 
-### Excerpt from working env:
+#### EL6 Grub Configuration
```
Comment (jeefberkey) [inline README.md]: legacy grub for more correctness

**Review by jeefberkey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nick-markowski) [reply README.md:68]: The script was for proof of concept, for now I'm not going to optimize a script that does not work.

**Review by jeefberkey — APPROVED** [review]

**Pull request merged**